articles/tales-from-fleet-security-securing-1password.md

```diff
@@ -0,0 +1,81 @@
+# Tales from Fleet security: securing 1Password
+
+![Securing 1Password](http://fleetdm.com/images/articles/securing-1password-1600x900@2x)
+
+No matter how much we wish SaaS tools had support for Single Sign-On (SSO), there are still so many
+websites and other resources that require individual passwords. Running a company without providing
+employees with a password manager is downright dangerous.
```

Comment:
> Running a company without providing employees with a password manager is downright dangerous

This statement sounds scary. I'd say FUDy, but I'm not sure if that's a proper adjective. Are you able to add a short sentence or two to back up your argument? There's a lot of chatter online about whether or not password managers are safe to use. I wonder if we could take the opportunity to debunk that myth and say why we chose to use a password manager at Fleet?